src/shared/components/barGraph/BarGraph.tsx

```diff
@@ -129,74 +137,84 @@ export default class BarGraph extends React.Component<IBarGraphProps, {colors: s
             .sort((a, b) => b.caseCount - a.caseCount)
             .slice(0, 20);
 
-        Highcharts.chart(this.chartTarget, {
-
-            chart: {type: 'bar', backgroundColor:'' },
-
-            exporting: { enabled:false },
-
-            title: {text: ''},
-
-            xAxis: {categories: cancerTypeStudiesArray.slice(0, 20).map((cancer) => (cancer.shortName)),
-                    labels: {style: {fontSize: '10px'}}},
-
-            yAxis: {
-                allowDecimals: false,
-                min: 0,
-                // max: 6000,
-                title: {text: ''}
-            },
+        const datasets = _.flattenDeep(cancerTypeStudiesArray.map((cancerStudySet, i) => (
+            cancerStudySet.studies.sort((a: ICounterObj, b:ICounterObj) => +b.allSampleCount - +a.allSampleCount).map((cancerStudy: CancerStudy, j: number) => {
```

Comment:
had to look that syntax + syntax up.  i don't think this should be necessary.  what else could these counts be beside numbers?   also, where is ICounterObj declared?